core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -279,6 +277,40 @@ public List<ActiveScan> getActiveScans(String tserver)
     }
   }
 
+  @Override
+  public List<ActiveScan> getActiveScans(ServerId server)
+      throws AccumuloException, AccumuloSecurityException {
+
+    @SuppressWarnings("unused")
+    var unused = Objects.nonNull(server);
+    Preconditions.checkArgument(server.getType() == ServerTypeName.SCAN_SERVER
+        || server.getType() == ServerTypeName.TABLET_SERVER);
```

Comment:
Not sure the Preconditions syntax is correct, attempted to add an error message.


```suggestion
    Preconditions.checkArgument(server.getType() == ServerTypeName.SCAN_SERVER
        || server.getType() == ServerTypeName.TABLET_SERVER, "Server type %s is not %s or %s.", server.getType(), ServerTypeName.SCAN_SERVER, ServerTypeName.TABLET_SERVER);
```

Reply:
Suggestion implemented in ebd6763